src/styles/common.scss

```diff
@@ -58,6 +58,7 @@
 }
 
 .button:disabled {
+    border-color: transparent;
```

Comment:
I am not sure how the disabled button should look like. But out of stock button looks like this for sure:

<img width="394" alt="Screenshot 2024-09-24 at 16 22 43" src="https://github.com/user-attachments/assets/e527e944-49aa-49a1-a152-ae2b1011303d">

So I removed border when a button is disabled.